Automattic/wp-calypso
# Stepper framework: Add import flow

#### Changes proposed in this Pull Request

This PR contains the first part of the effort introducing the whole import onboarding experience to the stepper framework.

- 🟢 `start/importer/capture` (signup) -> `setup/import` (stepper)
- ⚪️ `start/from/importing/{importer}` (signup) -> `setup/importer/{importer}`

* Registered an `import` flow inside a new non-linear stepper framework.
* Adjusted the stepper framework allowing consumption of the main redux store since the import module depends on it.
* Setup import flow to consume existing import module screens from the linear signup framework.

**Next steps** (not covered by this PR):
- Move the place where the import module lives. From the signup steps to `calypso/blocks`
- Register an importer flow equivalently with `/start/from/{importer}`

#### Testing instructions

* Go to the `setup/intent?siteSlug={SLUG}`
* Select `Import your site content` option
* Enter any URL you want to import from
* Check if it works in the same way as it is the case when you start from `/start/setup-site/intent?siteSlug={SLUG}`
* Check navigation `Back` and `I don't have a site address` header's navigation links
* In other words, it should be the same as it is on the `/start` signup flow

Related to #62238

Additonal information: pcmemI-u6-p2
